<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managevolumeheader</class>
 <widget class="QDialog" name="managevolumeheader">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>backup volume header</string>
  </property>
  <widget class="QLineEdit" name="lineEditBackUpName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelBackUpHeader">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>backup name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenFolder">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCreate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>c&amp;reate</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelDevicePath">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>volume path</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDevicePath">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFile">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPartition">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEditPassWord">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pBKeyFile">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbKey">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>key</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbKeyFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>keyfile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbNormalTrueCrypt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>normal volume</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbSystemTrueCrypt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>windows system volume</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbFDETrueCrypt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>whole drive encrypted volume</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbLuksHeader">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>manage a luks header</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbTrueCryptHeader">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>manage a truecrypt header</string>
    </property>
   </widget>
  </widget>
  <zorder>lineEditBackUpName</zorder>
  <zorder>labelBackUpHeader</zorder>
  <zorder>pbOpenFolder</zorder>
  <zorder>pbCreate</zorder>
  <zorder>pbCancel</zorder>
  <zorder>labelDevicePath</zorder>
  <zorder>lineEditDevicePath</zorder>
  <zorder>pushButtonFile</zorder>
  <zorder>pushButtonPartition</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>rbNormalTrueCrypt</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEditDevicePath</tabstop>
  <tabstop>pushButtonFile</tabstop>
  <tabstop>pushButtonPartition</tabstop>
  <tabstop>lineEditBackUpName</tabstop>
  <tabstop>pbOpenFolder</tabstop>
  <tabstop>rbLuksHeader</tabstop>
  <tabstop>rbTrueCryptHeader</tabstop>
  <tabstop>rbNormalTrueCrypt</tabstop>
  <tabstop>rbSystemTrueCrypt</tabstop>
  <tabstop>rbFDETrueCrypt</tabstop>
  <tabstop>rbKey</tabstop>
  <tabstop>rbKeyFile</tabstop>
  <tabstop>lineEditPassWord</tabstop>
  <tabstop>pBKeyFile</tabstop>
  <tabstop>pbCreate</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
